app/src/main/java/com/quran/labs/androidquran/ui/PagerActivity.java

```diff
@@ -799,12 +801,14 @@ public void onNewIntent(Intent intent) {
     }
   }
 
+  @Override
   public void jumpTo(int page) {
     Intent i = new Intent(this, PagerActivity.class);
     i.putExtra("page", page);
     onNewIntent(i);
   }
 
+  @Override
```

Comment:
since this method is override from interface, I think it is better to group it at the top class, below override methods from Activity.
Moreover this class should be re arranged so all public method at the top and all private method at the bottom 🤔